### ISSUE TITLE
`git-lfs version` fails if called in the root directory

### ISSUE THREAD

**Issue by larsxschneider** [github/git-lfs#1496]
`git-lfs version` fails on Windows (using git-bash) and OS X if called in the root directory.

**Versions tested:**
```
1.4.1
```

**Steps to reproduce:**
```
cd / && git-lfs version
```

**Output:**
```
panic: Error trying to init LocalStorage: mkdir lfs: permission denied

goroutine 1 [running]:
panic(0x377740, 0xc420102920)
       	/usr/local/Cellar/go/1.7/libexec/src/runtime/panic.go:500 +0x1a1
github.com/github/git-lfs/localstorage.ResolveDirs()
       	/private/tmp/git-lfs-20160826-6787-1o15v20/git-lfs-1.4.1/src/github.com/github/git-lfs/localstorage/currentstore.go:39 +0x4c3
github.com/github/git-lfs/commands.resolveLocalStorage(0xc420123d40, 0x5f2ae0, 0x0, 0x0)
       	/private/tmp/git-lfs-20160826-6787-1o15v20/git-lfs-1.4.1/src/github.com/github/git-lfs/commands/commands.go:342 +0x14
github.com/github/git-lfs/vendor/github.com/spf13/cobra.(*Command).execute(0xc420123d40, 0x5f2ae0, 0x0, 0x0, 0xc420123d40, 0x5f2ae0)
       	/private/tmp/git-lfs-20160826-6787-1o15v20/git-lfs-1.4.1/src/github.com/github/git-lfs/vendor/github.com/spf13/cobra/command.go:474 +0x255
github.com/github/git-lfs/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4200f64e0, 0xc42004fed0, 0x1)
       	/private/tmp/git-lfs-20160826-6787-1o15v20/git-lfs-1.4.1/src/github.com/github/git-lfs/vendor/github.com/spf13/cobra/command.go:551 +0x380
github.com/github/git-lfs/commands.Run()
       	/private/tmp/git-lfs-20160826-6787-1o15v20/git-lfs-1.4.1/src/github.com/github/git-lfs/commands/commands.go:70 +0x180
main.main()
       	/private/tmp/git-lfs-20160826-6787-1o15v20/git-lfs-1.4.1/git-lfs.go:33 +0x111
```

I think the problem was introduce here:
https://github.com/github/git-lfs/pull/1226

/cc @sinbad

**Comment by sinbad** [comment]
I think this is the same core issue as https://github.com/github/git-lfs/issues/1431, in that `version` does not need to resolve the local storage dirs. 

I *could* fix this right now:

```diff
diff --git a/commands/command_version.go b/commands/command_version.go
index d588566..180d313 100644
--- a/commands/command_version.go
+++ b/commands/command_version.go
@@ -20,9 +20,8 @@ func versionCommand(cmd *cobra.Command, args []string) {
 func init() {
        RegisterSubcommand(func() *cobra.Command {
                cmd := &cobra.Command{
-                       Use:    "version",
-                       PreRun: resolveLocalStorage,
-                       Run:    versionCommand,
+                       Use: "version",
+                       Run: versionCommand,
                }

                cmd.Flags().BoolVarP(&lovesComics, "comics", "c", false, "easter egg")
```

However @technoweenie will already be changing this code as part of #1495 so I'll put a note there instead.

**Comment by larsxschneider** [comment]
I just observed something in addition:
`git lfs version` seems to create a `lfs` directories (containing `objects` and `tmp`) in whatever directory you invoke the call.

**Comment by sinbad** [comment]
Yeah, that's the core of it - the fix I've proposed above will resolve that.

**Comment by technoweenie** [comment]
That's a good fix (that I'll take care of in #1495), but I wonder if we should worry about the panic from running `localstorage.ResolveDirs()` on `/`.

**Comment by sinbad** [comment]
It's deliberate in the code:
```
	if err != nil {
		panic(fmt.Sprintf("Error trying to init LocalStorage: %s", err))
	}
```
Do you mean it shouldn't be a panic & just an error message instead?

**Comment by ttaylorr** [comment]
I think we should still bail out of running LFS if we can't initialize LocalStorage (and we wanted to), but it doesn't need to be a panic 👍

**Comment by technoweenie** [comment]
Fixed in #1495:

```bash
$ cd /
$ git lfs version
git-lfs/1.4.1 (GitHub; darwin amd64; go 1.7; git a06acca)
```

Also, tweaking the error message in #1499.
